### PR TITLE
Update get-started.mdx

### PR DESCRIPTION
**Pull request by swarajbachu**
the previous command wasnt actually creating it to the framework as next, changing the tag is initializing the next project

### Summary

the command was about starting a new next project, but it is instead just running to create a normal project

### Screenshots (optional)
previous

Screenshot 2024-08-20 at 8 35 44 PM

### CODE REVIEW

**Comment by swarajbachu** [comment]
hello @IgorMinar can you look into this once

**Comment by dario-piotrowicz** [comment]
Thanks for the PR @swarajbachu 🙂 

Could you please update your PR to use the `PacakgeManager` starlight component (see [starlight-package-managers](https://github.com/HiDeoo/starlight-package-managers)) we're now using across the docs?

Something like:
```
<PackageManagers type="create" pkg="cloudflare@latest" args="--framework=next" />
```

That will make sure that we do get the right command for the various package managers 🙂 

PS: if you could apply the change also to all the other guides under `src/content/docs/pages/framework-guides` that would be really really appreciated! 😁 🫶

**Comment by swarajbachu** [comment]
sure, lemme do that

**Comment by swarajbachu** [comment]
> starlight-package-managers

just one thing to note is that this means it will appear 
this way

```
npm create cloudflare@latest -- --framework=next
```

 instead of 

```
npm create cloudflare@latest my-next-app -- --framework=next
```

or I can do 
```
<PackageManagers type="create" pkg="cloudflare@latest my-next-app" args="--framework=next" />
```
but it doesn't seem to be a good practice 


so for now I am doing the way you told me to do, lemme know if you need to change it further

**Comment by swarajbachu** [comment]
@dario-piotrowicz updated the way you mentioned for now

**Comment by dario-piotrowicz** [comment]
@swarajbachu thanks so much for updating the guides, it's really very much appreciated!

As you pointed out in your comment my suggestion wasn't actually fully correct (I did it on the spot as an example without checking things too much, sorry for the confusion)

I think that the right change would then be:
```ts
<PackageManagers
	type="create"
	pkg="cloudflare@latest"
	args="my-next-app --framework=next"
/>
```

This does seem to produce the correct results:
Screenshot 2024-09-05 at 12 30 38
Screenshot 2024-09-05 at 12 30 45


 and it's not, as far as I can tell, bad practice or anything like that.

What do you think? could you make this one last change? (to all the guides as well 😅) 🙏

**Comment by swarajbachu** [comment]
> @swarajbachu thanks so much for updating the guides, it's really very much appreciated!
> 
> As you pointed out in your comment my suggestion wasn't actually fully correct (I did it on the spot as an example without checking things too much, sorry for the confusion)
> 
> I think that the right change would then be:
> 
> ```ts
> <PackageManagers
> 	type="create"
> 	pkg="cloudflare@latest"
> 	args="my-next-app --framework=next"
> />
> ```
> 
> This does seem to produce the correct results: Screenshot 2024-09-05 at 12 30 38 Screenshot 2024-09-05 at 12 30 45
> 
> and it's not, as far as I can tell, bad practice or anything like that.
> 
> What do you think? could you make this one last change? (to all the guides as well 😅) 🙏

hmm yeah cool, now I am will do that, and do you want me to also add `bun`?

**Comment by dario-piotrowicz** [comment]
> hmm yeah cool, now I am will do that

@swarajbachu thanks a lot! ❤️ 

> and do you want me to also add `bun`?

I'm not sure 😕 (I guess it depends how much extra code that would entail)

I'd avoid that for now, the best person to ask would be @KianNH, but he's currently on holiday I believe

I'm not sure if Kian has already considered/looked into bun, to avoid delays with merging this PR I would just avoid adding bun and potentially think of such addition as a followup 🙂

**Comment by swarajbachu** [comment]
> > hmm yeah cool, now I am will do that
> 
> @swarajbachu thanks a lot! ❤️
> 
> > and do you want me to also add `bun`?
> 
> I'm not sure 😕 (I guess it depends how much extra code that would entail)
> 
> I'd avoid that for now, the best person to ask would be @KianNH, but he's currently on holiday I believe
> 
> I'm not sure if Kian has already considered/looked into bun, to avoid delays with merging this PR I would just avoid adding bun and potentially think of such addition as a followup 🙂

cool lemme do the remaining changes then

**Comment by swarajbachu** [comment]
@dario-piotrowicz  done dude

**Comment by swarajbachu** [comment]
hello dude @dario-piotrowicz

**Comment by dario-piotrowicz** [comment]
@swarajbachu sorry I was unable to review the PR yesterday 🙇, I'm having a look right now 🙂

File: src/content/docs/pages/framework-guides/deploy-a-hexo-site.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ pcx_content_type: how-to
 title: Hexo
 ---
 
-import { Render } from "~/components";
+import { Render, PackageManagers } from "~/components";
```
Comment (dario-piotrowicz) [inline src/content/docs/pages/framework-guides/deploy-a-hexo-site.mdx]:
```suggestion
import { Render } from "~/components";
```

we shouldn't import the component if we're not using it

Comment (dario-piotrowicz) [reply src/content/docs/pages/framework-guides/deploy-a-hexo-site.mdx]: PS: you can update the installation instructions below if you want and use the `packageManager` component, but I'd skip that in this PR not to keep increasing its scope 😅

File: src/content/docs/pages/framework-guides/deploy-a-svelte-site.mdx [hunk 2]
```diff
@@ -17,9 +17,11 @@ Use the [`create-cloudflare`](https://www.npmjs.com/package/create-cloudflare) C
 
 To use `create-cloudflare` to create a new Svelte project, run the following command:
 
-```sh
-npm create cloudflare@latest my-svelte-app -- --framework=svelte
-```
+<PackageManagers
+	type="create"
+	pkg="cloudflare@latest"
+	args="my-svelte-app --framework=sveltekit"
```
Comment (dario-piotrowicz) [inline src/content/docs/pages/framework-guides/deploy-a-svelte-site.mdx]:
```suggestion
	args="my-svelte-app --framework=svelte"
```

File: src/content/docs/pages/framework-guides/deploy-a-vite3-project.mdx [hunk 3]
```diff
@@ -3,17 +3,15 @@ pcx_content_type: how-to
...
 [Vite](https://vitejs.dev) is a next-generation build tool for front-end developers. With [the release of Vite 3](https://vitejs.dev/blog/announcing-vite3.html), developers can make use of new command line (CLI) improvements, starter templates, and [more](https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md#300-2022-07-13) to help build their front-end applications.
 
 Cloudflare Pages has native support for Vite 3 projects. Refer to the blog post on [improvements to the Pages build process](https://blog.cloudflare.com/cloudflare-pages-build-improvements/), including sub-second build initialization, for more information on using Vite 3 and Cloudflare Pages to optimize your application's build tooling.
 
 In this guide, you will learn how to start a new project using Vite 3, and deploy it to Cloudflare Pages.
 
-```sh
-npm create vite@latest
-```
+<PackageManagers type="create" pkg="create-vite@latest" />
```
Comment (dario-piotrowicz) [inline src/content/docs/pages/framework-guides/deploy-a-vite3-project.mdx]:
```suggestion
<PackageManagers type="create" pkg="vite@latest" />
```

File: src/content/docs/pages/framework-guides/deploy-an-analog-site.mdx [hunk 4]
```diff
@@ -15,25 +21,11 @@ In this guide, you will create a new Analog application and deploy it using Clou
...
-
-```sh
-bun create cloudflare@latest my-analog-app --framework=analog
-```
-
-</TabItem> </Tabs>
+<PackageManagers
+	type="create"
+	pkg="cloudflare@latest"
+	args="my-analog-app --framework=angular"
```
Comment (dario-piotrowicz) [inline src/content/docs/pages/framework-guides/deploy-an-analog-site.mdx]:
```suggestion
	args="my-analog-app --framework=analog"
```

**Review by dario-piotrowicz — COMMENTED** [review]
Fantastic job @swarajbachu, thanks so very much for the various updates! 🫶 

I've left a few comments for things that need amending, besides these small changes it looks great to me 😄

**Review by kodster28 — APPROVED** [review]

**Comment by kodster28** [comment]
> Fantastic job @swarajbachu, thanks so very much for the various updates! 🫶
> 
> I've left a few comments for things that need amending, besides these small changes it looks great to me 😄

Thanks for the contribution, @swarajbachu, I'll merge as soon as checks pass. Definitely appreciate the help here making sure other folks don't run into the issue you experienced.

**Pull request merged**